src/games/strategy/triplea/ui/RouteOptimizer.java

```diff
@@ -105,7 +96,9 @@ private Point normalizePoint(Point point) {
    * @return A List of all possible Points depending in map Properties
    *         size may vary
    */
-  private List<Point2D> getPossiblePoints(Point2D point) {
+  public List<Point2D> getPossiblePoints(Point2D point) {
+    mapWidth = mapPanel.getImageWidth();
```

Comment:
This assignment is only needed when tested, since this Method is normally called after those values are set in the getTranslatedRoute method.
I recall that this was used, when I was still calculating the route based on the camera position. I will change that